### ISSUE TITLE
add test to show Mustache-style tags within README code blocks work

### ISSUE THREAD

**Issue by ashleygwilliams** [npm/marky-markdown#160]
based on a previous issue, let's write a test to show that we *are* doing the right thing
------
https://github.com/npm/newww/issues/1750 from @passcod 

It appears that the Markdown renderer incorrectly strips out Mustache-style double-curlies in code blocks. Compare:

### Markdown source
https://raw.githubusercontent.com/Marak/faker.js/master/Readme.md

```
``` js
console.log(faker.fake('{{name.lastName}}, {{name.firstName}} {{name.suffix}}'));
// outputs: "Marks, Dean Sr."
```
```

(Using [Greek varias](https://en.wikipedia.org/wiki/Ancient_Greek_accent) to replace backticks there.)

### GIthub readme
https://github.com/Marak/Faker.js

![screenshot from 2016-01-15 10-38-53](https://cloud.githubusercontent.com/assets/155787/12338553/0a1dc046-bb75-11e5-8f54-2fe4428f7857.png)

### NPM readme
https://www.npmjs.com/package/faker

![screenshot from 2016-01-15 10-39-23](https://cloud.githubusercontent.com/assets/155787/12338571/17405324-bb75-11e5-8d17-87db4ccf7b4c.png)

**Comment by revin** [comment]
I've looked at this one before; seems like a newww problem with the template rendering...?

Running the markdown above through marky via CLI renders it as:

```html
<h1 id="user-content-test" class="deep-link"><a href="#test">test</a></h1>
<div class="highlight js"><pre class="editor editor-colors"><div class="line"><span class="source js"><span class="entity name type object js firebug"><span>console</span></span><span class="support function js firebug"><span>.log</span></span><span class="meta brace round js"><span>(</span></span><span>faker</span><span class="meta delimiter method period js"><span>.</span></span><span>fake</span><span class="meta brace round js"><span>(</span></span><span class="string quoted single js"><span class="punctuation definition string begin js"><span>&apos;</span></span><span>{{name.lastName}},&#xA0;{{name.firstName}}&#xA0;{{name.suffix}}</span><span class="punctuation definition string end js"><span>&apos;</span></span></span><span class="meta brace round js"><span>)</span><span>)</span></span><span class="punctuation terminator statement js"><span>;</span></span></span></div><div class="line"><span class="source js"><span class="comment line double-slash js"><span class="punctuation definition comment js"><span>//</span></span><span>&#xA0;outputs:&#xA0;&quot;Marks,&#xA0;Dean&#xA0;Sr.&quot;</span><span>&#xA0;</span></span></span></div></pre></div>
```

which renders in my browser as:

![screen shot 2016-03-01 at mar 1 9 13 41 am](https://cloud.githubusercontent.com/assets/94850/13435197/f47357dc-df8d-11e5-8ede-3ed74ae56a49.png)

**Comment by ashleygwilliams** [comment]
closed by #161
